README.md

```diff
@@ -461,6 +461,13 @@ sequencer.insertSteps({
 ```
 return value: **`sequencer`** (To allow method chaining)
 
+## Meta Modules
+
+Image sequencer supports saving a sequence of modules and their associated settings(a stringified sequence) as a `meta-module`. These meta-modules are saved in the local storage inside the browser and inside a json file in node.js. Meta-modules can be saved in node context using the CLI option
+```shell
+--save-as-meta-module "name stringified-sequence"
+```
+Sequencer supports a function `sequencer.saveMetaModule(name,sequenceString)` which saves the given string as a meta module. The function `sequencer.loadModules()` reloads the modules and meta modules into `sequencer.modules` and `sequencer.metaModules`
```

Comment:
> Image Sequencer can create/instantiate a new meta module with the function `sequencer.saveMetaModule(name, sequenceString)`, which returns a new module **containing** the other modules. 

Maybe also mention whether meta-modules can contain other meta modules?

And tell people to submit new meta-modules they create by opening an issue - they could be imported in or listed in the readme.

Reply:
Done!!